index.html

```diff
@@ -58,86 +64,104 @@ <h2>Glad to see you!</h2>
+    <div class="snapshot-desktop"><img src="dektopimages/Snapshoot Portfolio3.png" alt="snapshot"></div>
     <div class="individual-card">
-      <div><h2>Tonic</h2></div>
+      <div class="mobile"><h2>Tonic</h2></div>
+      <div class="desktop"><h2>Facebook 360</h2></div>
       <ul class="li-elements">
-        <li class="current-li">Canopy</li>
+        <li class="current-li1">Canopy</li>
+        <li class="desktop">Facebook</li>
         <li><img class="counter" src="snapshotimages/Counter.png" alt="counter"></li>
-        <li>Back End Dev</li>
+        <li class="mobile2">Back End Dev</li>
+        <li class="desktop2">Full Stack Dev</li>    
         <li><img class="counter" src="snapshotimages/Counter.png" alt="counter"></li>
         <li>2015</li>
       </ul>
       <div><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
       <ul class="languages">
         <li><button class="language-button">html</button></li>
+        <li class="desktop"><button class="language-button">Ruby on Rails</button></li>
         <li><button class="language-button">css</button></li>
         <li><button class="language-button">javascript</button></li>
       </ul>
       <div><button class="see-project-button">See Project</button></div>
     </div>
   </div>
 
-  <div class="parent-container">
-    <div class="snapshot"><img src="snapshotimages/Snapshoot Portfolio (3).png" alt="snapshot"></div>
+  <div class="parent-container2">
+    <div class="snapshot-upper"><img src="snapshotimages/Snapshoot Portfolio (3).png" alt="snapshot"></div>
     <div class="individual-card">
-      <div><h2>Multi-Post Stories</h2></div>
+      <div class="mobile"><h2>Multi-Post Stories</h2></div>
+      <div class="desktop"><h2>Uber Navigation</h2></div>
+      
       <ul class="li-elements">
         <li class="current-li">Canopy</li>
         <li><img class="counter" src="snapshotimages/Counter.png" alt="counter"></li>
-        <li>Back End Dev</li>
+        <li class="mobile2">Back End Dev</li>
+        <li class="desktop2">Lead Developer</li>
         <li><img class="counter" src="snapshotimages/Counter.png" alt="counter"></li>
         <li>2015</li>
       </ul>
       <div><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
       <ul class="languages">
         <li><button class="language-button">html</button></li>
+        <li class="desktop"><button class="language-button">Ruby on Rails</button></li>
         <li><button class="language-button">css</button></li>
         <li><button class="language-button">javascript</button></li>
       </ul>
       <div><button class="see-project-button">See Project</button></div>
     </div>
+    <div class="snapshot-lower"><img src="dektopimages/Snapshoot Portfolio4.png" alt="snapshot"></div>
   </div>
   </section>
 
   <section class="about-myself">
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 Most of the elements in your `about-myself` section have an error in their positioning, so when you change the `width` value to the desired values, they are not positioned responsively and create unnecessary spaces too. Please correct them🙏🙏🙏 

- ### Your design ❌ 

![image](https://user-images.githubusercontent.com/63915024/199765223-1a5ac456-827b-4509-abb1-c43fb24d82b8.png)

- ### Figma design ✔️ 

![image](https://user-images.githubusercontent.com/63915024/199765286-3cdb0ef1-b9bb-4ae4-aac5-000b8fad63c8.png)

Reply:
Fixed.